.github/workflows/remote-copy.yml

```diff
@@ -157,20 +178,27 @@ jobs:
         id: copy
         env:
           REMOTE_RSYNC_FILE_LIST_PATH: ${{ vars.REMOTE_TMP_DIR }}/remote-copy-files-${{ github.run_id }}.log
+          LOCAL_RSYNC_FILE_LIST_PATH: ./files-copied.log
         # In this step, we rsync the files from the source to the target,
         # capturing the list of files copied as output of the step, since
         # it's used in the manifest step.
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
-          files=$(rsync --recursive --out-format="%n" \
+          rsync --recursive --out-format="${{ inputs.target }}/%n" \
               ${{ ! inputs.overwrite-target && '--ignore-existing' || '--update' }} \
               ${{ inputs.source }} ${{ inputs.target }} \
-            | tr '\n' ' ' \  # convert to space-separated for output
+            | grep --invert-match '/$' \
+            | tr '\n' ' ' \
             > ${{ env.REMOTE_RSYNC_FILE_LIST_PATH }}
```

Comment:
Why not use rsync's [in-built logging function](https://serverfault.com/a/703847) and then grab the list of file paths from the log?

Reply:
Mostly because we are editing the output of `rsync` before piping it to a file (the `grep --invert-match '/$'` for removing copied directories (which have `/` at the end), and `tr '\n' ' '` for converting it to space-separated). I didn't want to create two files for the rsync-logged output and then another for the processing that we need to do on that file. Happy to change it if its more maintainable the other way, though.

Reply:
All good. You have reasons. Happy to go with that.